client/src/stylesheets/components/_Tools.scss

```diff
@@ -0,0 +1,57 @@
+.tool-panel-wrapper {
```

Comment:
this might need to change depending on what happens to the ToolsList component

Reply:
This has been renamed to ToolList.scss.

The file now has only the scss needed to style the CollpasePanel body in the ToolList jsx